app/services/withdraw_participant.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+class WithdrawParticipant
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveModel::Validations::Callbacks
+
+  attribute :cpd_lead_provider
+  attribute :participant_id
+  attribute :reason
+  attribute :course_identifier
+  attribute :force_training_status_change
```

Comment:
Could you remove this attribute as we don't need it since we will be using `Induction::ChangeInductionRecord`?

Reply:
Hi @cwrw tksss 👍 😄  it's been addressed in [51b8090](https://github.com/DFE-Digital/early-careers-framework/pull/2639/commits/51b8090cf4b44933d43fcf8e0ccc693fc7ae0838)